src/System.Device.Gpio/System/Device/Gpio/SafeChipHandle.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace System.Device.Gpio
+{
+    class SafeChipHandle : SafeHandle
```

Comment:
Did we intend this to be public? I would also probably define this class on the Interop side instead of the managed code. @tarekgh thoughts?

Reply:
Moving to Interop side, also will make it internal